rs/macros/core/src/service/mod.rs

```diff
@@ -529,7 +529,7 @@ impl<'a> HandlerGenerator<'a> {
     }
 
     fn result_type(&self) -> Type {
-        self.handler.result().clone()
+        type_to_meta_type_with_static_lifetime(self.handler.result().clone())

```

Comment:
shared::func_name
is more ideomatic :-)

Reply:
can we rename it into something like
`replace_any_lifetime_with_static_one`
Even though it is long, but it is shorter then the current name and more descriptive to my POV